bql/planner/data_access.go

```diff
@@ -570,10 +580,10 @@ func tripleToRow(t *triple.Triple, cls *semantic.GraphClause) (table.Row, error)
 			return nil, nil
 		}
 	}
-
 	if cls.PAnchorAlias != "" {
 		if p.Type() != predicate.Temporal {
-			return nil, fmt.Errorf("cannot retrieve the time anchor value for non temporal predicate %q in binding %q", p, cls.PAnchorAlias)
+			// in the case of AT binding for an immutable predicate we just want (for now) to skip this triple and proceed to the next one, not returning any errors.
```

Comment:
nit: the "for now" might be confusing because doesn't specify a fixed time period. Does it means that this behavior is temporal? Should we add a TODO with more explanation for a next PR or maybe just drop the "for now"?

Reply:
Hi Jose, thank you for the comments. Yes, it means that this behavior is temporal and it is indeed changed together with this comment in a next PR, that I have just finished (PR #127). 
But I agree that it is simpler and clearer to just drop this `"(for now)"` from this comment here at the moment. I have just done so. 🙂